src/piadm.sh

```diff
@@ -359,6 +359,13 @@ install() {
 		tar -cf - -C "${tdir}/mnt/boot" . | \
```

Comment:
As I said in the man page, I wonder if we should make the "common" directory as part of pool refresh or poll enabling.  Basically the refresh_or_disable_pool() function and the enablepool() function should have the ability to mkdir /$bootfs/common (and leave it to the user).